src/main/kotlin/app/revanced/patches/reddit/misc/disablescreenshotpopup/fingerprints/DisableScreenshotPopupFingerprint.kt

```diff
@@ -0,0 +1,15 @@
+package app.revanced.patches.reddit.misc.disablescreenshotpopup.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object DisableScreenshotPopupFingerprint : MethodFingerprint(
+    opcodes = listOf(
+        Opcode.AND_INT_LIT8,
+        Opcode.CONST_4,
+        Opcode.IF_NE
+    ),
+    customFingerprint = { methodDef, _ ->
+        methodDef.name == "invoke"
+    }
```

Comment:
This app probably has many methods that are named "invoke". You should add more safety guards